app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name, health=100, hidden=False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f'{{Name: {self.name}, ' \
+               f'Health: {self.health}, ' \
+               f'Hidden: {self.hidden}}}'
```

Comment:
Please replace `'` -> `"`

Reply:
Still not fixed :(
We prefer to use double quotes